python/MooseDocs/commands/build.py

```diff
@@ -152,6 +152,9 @@ def _init_large_media():
             log.info("Initializing the 'large_media' submodule for storing images above 1MB.")
             subprocess.call(['git', 'submodule', 'update', '--init', 'large_media'],
                             cwd=MooseDocs.MOOSE_DIR)
+            MooseDocs.PROJECT_FILES.update(MooseDocs.ls_files(
+                                           os.path.join(MooseDocs.MOOSE_DIR, 'large_media')))
```

Comment:
Remove. This will only add 'large_media' files to PROJECT_FILES if it is being initialized.

Reply:
@aeslaughter isn't this sort of what we want? Because, if `large_media` already exists, it would have been added in the `__init.py__`. This is the case where nothing came back in `__init.py__`, so here we're updating again, now that it has been checked out.

Reply:
You need to think about it with respect to the script that is running, which contains `import MooseDocs`, which is what calls the functions in this __init__.py file. PROJECT_FILES is a global variable that is created and updated each time you run the script, not just once.

The first time you run the script without large_media it will init the submodule and create the global variable PROJECT_FILES. The second time, the init part will not run and the global variable will not contain the files.

Reply:
@aeslaughter I see. Well I implemented the changes you asked for. I deleted the `init_large_media()` function in main.py. But I only now just realized were not even using the get_large_media.sh script anymore... let me know what you think of this